hw/bsp/rb-blend2/include/bsp/boards.h

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
Ah I blindly copied it from rb-nano2... so I meant nothing other than the license comment already included in this file.